### DISCUSSION TITLE
Ability to search entries in a relationship form by custom fields

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#3156] [Feature Requests]
Right now the only way to find an entry in a relationship form is by its title:

<img width="315" alt="image" src="https://user-images.githubusercontent.com/12234507/227439916-bcdf0430-8c41-4a0d-b274-113a3e04330f.png">

But there are many cases, where it could be very useful to search entries also by other fields. Can we add a checkbox "Search all fields" here? Or vice versa, "Search titles only", like in an entry manager.

**Comment by ic360-com** [comment]
Nice idea.
It would be very cool, if instead of a single search field being presented here, a simplified version of the "Entry List Filter" bar was available.
This would be especially useful where there are very few pre-defined limitations set up in the Relationship field... and there ends up being a very long list of Entries available.
